db/migrate/20230602034711_rename_everything_to_topics_link.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class RenameEverythingToTopicsLink < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec <<~SQL
+      UPDATE sidebar_urls su1
+      SET name = 'Topics', value = '/topics'
+      FROM sidebar_urls su2
+      INNER JOIN sidebar_section_links ON sidebar_section_links.linkable_id = su2.id
+      INNER JOIN sidebar_sections ON sidebar_sections.id = sidebar_section_links.sidebar_section_id AND sidebar_sections.section_type = 0
+      WHERE su1.id = su2.id AND su2.value = '/latest'
```

Comment:
Don't we have to check that `name == 'Everything'` as well since an admin might have already renamed this to something else?